### DISCUSSION TITLE
Can we give user permission to DAGs based on tags.

### DISCUSSION THREAD

**Discussion by intc-hharshtk** [apache/airflow#21963] [Q&A]
### Are you willing to submit a PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by jedcunningham** [comment]
Not based on tags, but you can set `access_control` on your DAG, which can achieve a similar result. For example:

```
access_control={"SomeRole": {"can_read"}},
```

**Reply by intc-hharshtk** [reply]
But then I have to explicitly give permission to each DAG.
